2019/f8801.py

```diff
@@ -10,8 +10,8 @@ def __init__(f, inputs, f1040, f6251):
             return
```

Comment:
Delete the "NOT FINAL" comment from the top of the file now that's you've finished the work.